polar-core/src/lexer.rs

```diff
@@ -108,6 +110,8 @@ impl ToString for Token {
             Token::Bang => "!".to_owned(),          // !
             Token::Mul => "*".to_owned(),           // *
             Token::Div => "/".to_owned(),           // /
+            Token::Mod => "mod".to_owned(),         // mod
+            Token::Rem => "rem".to_owned(),         // rem
```

Comment:
Why not `%` instead of `rem`?

Reply:
Because then it's ambiguous whether it means `rem` or `mod`. They agree on nonnegative integers, but for other cases they behave differently. This has been a perennial problem in languages; e.g., see [this crazy Wiki chart](https://en.wikipedia.org/wiki/Modulo_operation#In_programming_languages). I agree that `%` is slightly nicer, but this way we can avoid users asking, "ok, but does `%` mean `mod` or `rem`?"